### ISSUE TITLE
[Bug]: [a11y] 3 dot menu - Consider making "Bookmarks" clickable item larger

### ISSUE THREAD

**Issue by AdinaPetridean** [mozilla-mobile/fenix#27173]
### Steps to reproduce

1. Have Google Accessibility Scanner installed.
2. Navigate to any web page, and tap on the nav bar's three-dot menu.
3. Scan the 3 dot menu. 
4. Observe the scan.

### Expected behaviour

No suggestions are made.

### Actual behaviour

Consider making this clickable item larger. This item's height is 22 dp. Consider making the height of this touch target 48 dp or larger.

### Device name

Xperia 10 

### Android version

Android 12

### Firefox release type

Firefox Beta

### Firefox version

Firefox Beta 106.0b2

### Additional information

![bookmarks](https://user-images.githubusercontent.com/108326843/192305258-69fc6624-1954-402a-b9d3-906b66d4edaf.JPEG)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21671)

**Comment by bigConifer** [comment]
Duplicate of https://github.com/mozilla-mobile/fenix/issues/24560

@AdinaPetridean there is also a ticket on AC covering this bug, see https://github.com/mozilla-mobile/android-components/issues/11958#issuecomment-1087836598.
Moreover please properly check the recommendations of the Accessibility Scanner: The larger parent container performs the same action as selecting the Bookmarks item. So it should be considered making that item not clickable separately instead of increasing it's height.

**Comment by AdinaPetridean** [comment]
Based on the latest comments, I have marked this issue as CLOSED.
